gradle.properties

```diff
@@ -87,6 +87,7 @@ jaegerThriftVersion=0.31.0
 jakartaActivationVersion=1.2.2
 javaDiffUtilsVersion=4.5
 javassistVersion=3.24.1-GA
+javatuplesVersion=1.2
```

Comment:
Where are we using this new dependency? Sorry I didn't notice any usages

Reply:
I initially had a requirement to use this dependency, and later removed it. I'll remove it from here as well